src/Controller/SpaceController.php

```diff
@@ -13,11 +13,34 @@ class SpaceController extends AbstractController
 {
 
     /**
-     * @Route("/space/{location}", name="app_space_list")
+     * @Route("/", name="app_home")
+     */
+    public function search(Request $request): Response
+    {
+
+        $form = $this->createForm(SearchType::class, null, array('method' => 'GET'));
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            $location = $form->get('location')->getData();
+
+            return $this->redirectToRoute('app_space_list', [
+                'location' => $location,
+            ]);
```

Comment:
once mixed with index, no need to redirect: change the value of the rendered space list

Reply:
done